workflows/DATA_TO_MODEL.nf

```diff
@@ -3,14 +3,18 @@
 nextflow.enable.dsl = 2
 // Empty string default to avoid warning
 params.submissions = ""
+// The expected file type for the submissions
+params.file_type = "csv"
 // Synapse ID for Submission View
 params.view_id = "syn52576179"
-// Scoring Script
-params.scoring_script = "data_to_model_score.py"
-// Validation Script
-params.validation_script = "validate.py"
-// Testing Data
-params.testing_data = "syn51390589"
+// The container that houses the scoring and validation scripts
+params.challenge_container = "ghcr.io/jaymedina/jenny-test-evaluation:latest"
+// The command used to execute the Challenge scoring script in the base directory of the challenge_container: e.g. `python3 path/to/score.py`
+params.execute_scoring = "python3 /home/user/score.py"
+// The command used to execute the Challenge validation script in the base directory of the challenge_container: e.g. `python3 path/to/validate.py`
+params.execute_validation = "python3 /home/user/validate.py"
+// Synapse ID for the ground truth folder, which houses the ground truth file
+params.groundtruth_id = "syn65491926"
```

Comment:
NIT: It might help this script to be a little cleaner if these defaults were set in a testing profile or just in `nextflow.config` as plain defaults.

Reply:
Looking at the MTD workflow script there are a lot of defaults set both here and there that are the same.

Reply:
Good point! I do like leaving some of these values as "defaults" e.g. `execute_scoring`, to make it easier for users to contribute and only change necessary inputs. So how about I make 2 separate profiles -- `data_to_model_prototype` and the other -- where I put in the non-default params? i.e. `view_id` and so on.